tests/test_google_client.py

```diff
@@ -0,0 +1,51 @@
+from bittrade.data.google_client import get_close_price_table
```

Comment:
note I changed the location of your test file: all test files should be in this tests/ directory with names such as test_.. or .._test.py